addons/pos_restaurant/static/src/overrides/models/pos_store.js

```diff
@@ -357,11 +355,10 @@ patch(PosStore.prototype, {
     tableHasOrders(table) {
         return this.getActiveOrdersOnTable(table).length > 0;
     },
-    async transferOrder(destinationTable) {
-        const order = this.models["pos.order"].getBy("uuid", this.orderToTransferUuid);
+    async transferOrder(orderUuid, destinationTable) {
+        const order = this.models["pos.order"].find((o) => o.uuid === orderUuid);
```

Comment:
`getBy` didn't works anymore? 

If not, it should be cool to fix it instead of hidding the issue with a workaround.